### PR TITLE
Ignore some of the NHibernate methods

### PR DESCRIPTION
**Pull request by hazzik**

### CODE REVIEW

File: Source/AsyncGenerator.Tests/ExternalProjects/NHibernate/Fixture.cs [hunk 1]
```diff
@@ -150,6 +150,15 @@ private MethodConversion GetMethodConversion(IMethodSymbol symbol)
 					}
 					break;
 
+				case "PositionalParameterSpecification":
+				case "NamedParameterSpecification":
```
Comment (hazzik) [inline Source/AsyncGenerator.Tests/ExternalProjects/NHibernate/Fixture.cs]: @maca88 for some reason these 2 methods are not generated for my branch https://github.com/hazzik/nhibernate-core/tree/NH-2088

**Comment by maca88** [comment]
Will check why those two methods are not generated and also try to alternate the generator to not generate methods that are not used.

**Comment by hazzik** [comment]
>also try to alternate the generator to not generate methods that are not used

Nah, that's OK. The rules would be rather complicated.

**Comment by maca88** [comment]
[Fixed](https://github.com/maca88/AsyncGenerator/commit/bb792bd80ae10a81c15b1d426acc368cdf584586) the missing generation of those two methods. The problem lied in the `FindImplementationsAsync` method as will not find all implementations when there is an abstract method in between, only the abstract method is returned.

**Pull request merged**